client/src/pages/Login.js

```diff
@@ -116,6 +118,20 @@ export default function Login({ state, dispatch }) {
                 </Link>
               </Grid>
             </Grid>
+            <Grid container direction="row" justify="center" spacing={1}>
+              <Grid item>
+                <Typography variant="subtitle1">or just log in as a</Typography>
+              </Grid>
+              <Grid item>
+                <Link
+                  component={RouterLink}
+                  onClick={guestLogin}
+                  variant="subtitle1"
+                >
+                  {"Guest"}
+                </Link>
```

Comment:
Link component is showing an error on FE console because it does not have a **to** attribute. Probably using a button would solve this.

Reply:
or I can do to='#'